tests/QARegressionTests/test_Issues/test_CoreIssue119.py

```diff
@@ -0,0 +1,41 @@
+from business_rules.operators import DataframeType
+import pandas as pd
+import unittest
+import pytest
+
+"""This regression test is for accpetance criteria of Core Issue 119.
+These tests only checks if the is_unique_set and is_not_unique_set functions now
+handle the missing columns in the value list properly by ignoring them or not.
+For other tests please efer to the unit tests"""
+
+
+@pytest.mark.regression
+class TestCoreIssue119(unittest.TestCase):

```

Comment:
These tests are covered in unit tests and should be removed from regression tests

Reply:
removed